tensorboard/plugins/hparams/backend_context.py

```diff
@@ -190,9 +190,7 @@ def read_last_scalars(self, ctx, experiment_id, run_tag_filter):
             experiment_id=experiment_id,
             plugin_name=scalar_metadata.PLUGIN_NAME,
             run_tag_filter=run_tag_filter,
-            # TODO(#3436): We assume that downsampling always includes
-            # the most recent datum, which holds for all implementations
-            # of interest but is not yet required by the interface.
+            # Downsampling always includes the most recent datum.
```

Comment:
nit: This sounds a bit like this is specifically the case when downsampling. Maybe let's rephrase a bit?

This is a bit longer, but I think it makes it clearer: "read_scalars always includes the most recent datum, so down-sampling to one means fetching the latest value"

Up to you, either way is probably clear enough.